main.js

```diff
@@ -125,22 +137,22 @@ function addClicks() {
 
 }
 
-function getCoords(addr) {
+async function getCoords(addr) {
 
     let urlAddress = encodeURIComponent(addr);
-   
-    const coordObj = {};  
 
-    fetch(`https://nominatim.openstreetmap.org/search/?q=${urlAddress}&format=json`)
+    const coordObj = {};
+
+    let coords = await fetch(`https://nominatim.openstreetmap.org/search/?q=${urlAddress}&format=json`)
```

Comment:
here we await the fetch and assign it to a variable so that we can return info out of a Resolved Promise